qa/tasks/mgr/dashboard/test_osd.py

```diff
@@ -260,10 +267,114 @@ def test_add_osd_flag(self):
             'sortbitwise', 'recovery_deletes', 'purged_snapdirs', 'noout',
             'pause', 'pglog_hardlimit'
         ])
-        self.assertEqual(flags, sorted([
+        self.assertEqual(sorted(flags), sorted([
```

Comment:
```suggestion
        self.assertCountEqual(flags, [
```
[assertCountEqual](https://docs.python.org/3.6/library/unittest.html#unittest.TestCase.assertCountEqual ) ?

Reply:
Good suggestion!